### PR TITLE
Create Post “2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset”

### PR DESCRIPTION
**Pull request by jkgoodrich**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Using the gnomAD ancestry principal components analysis loadings and
+  random forest classifier on your dataset
+date: 2021-09-27
+order: 1
+authors:
+  - Julia Goodrich
+  - gnomAD Production Team
+---
+By popular request, we are now releasing the ancestry principal components analysis (PCA) variant loadings and accompanying random forest (RF) model used for global ancestry inference in gnomAD [v2](https://gnomad.broadinstitute.org/downloads#v2-ancestry-classification) and [v3](https://gnomad.broadinstitute.org/downloads#v3-ancestry-classification). This post discusses how those files were [generated](https://docs.google.com/document/d/1LrkQ3Cl21oU4irz_DX0WMw9gPLpz4IVANfaFc1JK87I/edit#heading=h.1ofxh16vagz9) and how they can be [used](https://docs.google.com/document/d/1LrkQ3Cl21oU4irz_DX0WMw9gPLpz4IVANfaFc1JK87I/edit#heading=h.wkprsvbvekdz) on another dataset. However, **the use of these resources will not be appropriate for all datasets**, and therefore we are including a discussion of the [caveats](https://docs.google.com/document/d/1LrkQ3Cl21oU4irz_DX0WMw9gPLpz4IVANfaFc1JK87I/edit#heading=h.g26t9x5c0a4l) associated with using these loadings and the RF model.
```
Comment (nawatts) [inline content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md:10]: There are several links to the draft Google Doc here.

Linking to sections within the page can be done by linking to the section heading's anchor. The anchor ID is the section heading in all lower case with spaces replaced by dashes.

For example, to link to the section with heading
```
### How to use the loadings and RF model on your own dataset
```
the link would be
```
[link text](#how-to-use-the-loadings-and-rf-model-on-your-own-dataset)
```

For another example, see the "Topics" section of the ["gnomAD v3.1 New Content, Methods, Annotations, and Data Availability" post](https://raw.githubusercontent.com/broadinstitute/gnomad-blog/main/content/posts/2020-10-gnomad-v3-1-new-content-methods-annotations-and-data-availability.md).

File: content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* When using the gnomAD loadings and RF model on a dataset that includes ancestries that are not well represented (or not represented at all) in gnomAD, **the RF model will likely mis-classify samples from those ancestries that are underrepresented or missing from gnomAD**. This is likely to particularly affect samples from regions that are very poorly sampled in gnomAD, such as Oceania, South-East Asia, the Middle East, and Africa, but will also affect other groups.
+
+* The workflow described [above](https://docs.google.com/document/d/1LrkQ3Cl21oU4irz_DX0WMw9gPLpz4IVANfaFc1JK87I/edit#heading=h.wkprsvbvekdz) will always return ancestry inference results, but **it is difficult to determine if the gnomAD RF model is actually doing an adequate job on ancestry inference in your specific dataset.** When we run this RF model for the gnomAD ancestry assignment, we can explicitly measure performance by holding back a certain percentage of samples with known ancestry labels from the training set and use them to evaluate the RF model classification.
```
Comment (nawatts) [inline content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md:65]: Another link to the Google doc.

File: content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md (hunk 1 above)
Comment (jkgoodrich) [reply content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md]: Thank you!!

**Review by nawatts — APPROVED** [review]

File: content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Using the gnomAD ancestry principal components analysis loadings and
+  random forest classifier on your dataset
+date: 2021-09-27
```
Comment (nawatts) [inline content/posts/2021-09-using-the-gnomad-ancestry-principal-components-analysis-loadings-and-random-forest-classifier-on-your-dataset.md:4]: This is out of date.

**Review by nawatts — CHANGES_REQUESTED** [review]
Sorry, just noticed one other thing.

**Pull request merged**